linux/gen_and_run_parallel.sh

```diff
@@ -3,9 +3,14 @@ num_of_runs=$1
 circle_consts=$2
 matrix_consts=$3
 simulation_consts=$4
-output_folder=$5
+TIMESTAMP=`date --utc -d "+3 hours" +_%d%m%Y_%H`
```

Comment:
Why do you use --utc and +3 ?
That is incorrect, Israel is GMT +2 (but sometime can also be +3)
Why not just use date +_%d%m%Y_%H

Reply:
because the time will be wrong if we don't add the 3 hours, you are correct about the clock change but i don't think we can solve it easily

Reply:
You can, do not use --utc

Reply:
the default time is the machine time, which is not israel's time

Reply:
Mhmmm I dont like this so much, but I guess it is OK for now :-/